app/src/main/java/org/lineageos/twelve/ui/recyclerview/SimpleListAdapter.kt

```diff
@@ -15,19 +15,19 @@ import androidx.recyclerview.widget.RecyclerView
 /**
  * A very basic ListAdapter that holds only one type of item.
  * @param diffCallback A [DiffUtil.ItemCallback] provided by the derived class
- * @param viewClass The [Class] of the [View], needed due to the restriction of reified on classes.
+ * @param factory The factory of the [View]
  *   This will be inflated using the constructor that takes just a [Context].
```

Comment:
Delete?

Reply:
- Ah I missed it, feel free to fix it up.
Now the unit has enough info